kiwi/config/functions.sh

```diff
@@ -998,11 +998,13 @@ function suseSetupProduct {
     # /etc/SuSE-brand or /etc/os-release or the latest .prod file
     # available in /etc/products.d
     # ----
+    set -e
```

Comment:
This changes the behavior on a global level. Any command with exit != 0 will now cause the script to exit. The setting will also persist beyond the scope of the method. We should not do this in a function imho

Reply:
Oh, I thought it would be only in this function scope. I'll remove it then.